app/services/matchers/party_name_splitter.rb

```diff
@@ -3,10 +3,9 @@ module Matchers
   class PartyNameSplitter
     attr_reader :party, :full_name, :split_name, :split_count
 
-    def initialize(party, full_name)
+    def initialize(party)
```

Comment:
removing the full_name parameter since party should already have a full_name

Reply:
yup